### DISCUSSION TITLE
Inject Script - main.js

### DISCUSSION THREAD

**Discussion by zikrome** [lencx/ChatGPT#400] [Q&A]
I have searched on here and can't find what to do with main.js for use with Inject Script?
The original main.js throws the error "console in undefined" - What can I add to main.js so that I can process a prompt automatically by running "Preferences" -> "Inject Script" ?

**Comment by lencx** [comment]
If you lack programming experience, please do not modify the main.js content randomly, it may lead to web page errors.

**Reply by zikrome** [reply]
I have over 10 years of programming experience. There is no guide as to what should go in main.js
Even the original main.js throws an error when you run inject script

**Reply by lencx** [reply]
Please provide screenshots, main.js and console (Help -> Toggle Developer Tools -> console).

**Reply by zikrome** [reply]
> Please provide screenshots, main.js and console (Help -> Toggle Developer Tools -> console).

I cleared console before beginning, ran inject script, and then checked console. Screenshots attached. I also updated first.

![console](https://user-images.githubusercontent.com/15665919/219621651-c752df4d-ecad-4f47-b86d-c5acf6bfce0f.png)
![main](https://user-images.githubusercontent.com/15665919/219621654-5a48dd96-ad87-40c1-929c-42a5be17ecc4.jpg)
